root/etc/e-smith/templates/etc/pam.d/cockpit/10base

```diff
@@ -0,0 +1,27 @@
+#%PAM-1.0
+auth	   required	pam_sepermit.so
+auth       substack     password-auth
+auth       include      postlogin
+auth       optional     pam_ssh_add.so
+# the root and members of domain admins are always allowed
+# if pam auth succeed, it ignores  pam_listfile.so
+auth [success=2 default=ignore] pam_succeed_if.so user in [root]
+auth [success=1 default=ignore] pam_succeed_if.so user ingroup [domain admins]
+# select group allowed to login to cockpit (root & domain admins alway OK)
+auth    required       pam_listfile.so \
+        onerr=fail  item=group  sense=allow  file=/etc/nethserver/authorizations/cockpit.access
+account    required     pam_nologin.so
+account    include      password-auth
+password   include      password-auth
+# select group allowed to login to cockpit (root & domain admins alway OK)
+auth    required       pam_listfile.so \
+        onerr=fail  item=group  sense=allow  file=/etc/nethserver/authorizations/cockpit.access
```

Comment:
Is this a repeated line?

Reply:
yes, truncated, you prefer one line ?

Reply:
No, the line break is fine. I wanted to say that the above lines 17-18 are equal to 11-12. They seem repeated, and should be removed IMO.

Reply:
oups got it, 16-17-18 must be removed it cannot work

Reply:
done